tsconfig.json

```diff
@@ -21,6 +21,6 @@
     "lib": ["dom", "esnext"],
     "preserveSymlinks": true
   },
-  "include": ["src/**/*.ts"],
+  "include": ["src/**/*.ts", "perf/**/*.ts", "perf/**/*.js"],
```

Comment:
I believe the lack of `perf/**/*.ts` here before is why ESLint complained about `enum`, and also didn't catch all of the missing annotation errors that you hinted at.

Reply:
FWIW, I checked `out/replicache.d.ts` after this change, and couldn't detect any hints of these new files in there, so I believe this is okay, but this is definitely beyond my ken.

Reply:
sgtm